.travis.yml

```diff
@@ -10,21 +10,33 @@ env:
   # Encrypted Coveralls token. The secret token for your repository can be found at the
   # bottom of your repository's page on Coveralls. To encrypt it, install the travis gem
   # and run:
-  #     travis encrypt -r ubuntu-core/snappy COVERALLS_TOKEN=$secret_token
-  - secure: JJEh1YfzNJGb1//ca447JtRunHmNXsVDGy5fS882/RQV3XtS9mCaSS+PZF4AbVABSiDw96mvwv05KmhdgV575y0NVHuRPd3rJBku9HOmhOKfCykn5tXgVJUEgeW5HIPusltRkXJUxnm48l+RfHHJU1h6iJnpGjvwxygeQljM4d22+G493ymvv4KZJSnbtuDbmhWvnZtdV2cCFeZ+EWqLgyWaFQmMgeMkYYcY3Cy19d++WquLs4woN9S9szrXJQlIv3xuZ8XNLbWmHoe6uD4y1Fn7YYfRgz861KqusxcGO9OyCh1S6uLgDuETS4Uw2zBg77vv59jSJieEfcY8jQtFE65axot/90jwhp5IV0tIpd9O/AMZXIGqKk44RU+Sj9gDbPbBIhj9dew3BTwSuOcNITSg1a4sOkxHGyIj2NozCkgFqPq0d4RM7xRrOYfnmjP1sJ+tEH4XMbs4gNZ/abccJXeUx6i5qWhDdnix132C62vvl+ItnsfMtkXQ34hluYxoab+rS0Zz3MM/ENqHySvU/R2n7+WTVXHyoK55HvExX7Y+McqPU2uVsHDabmDmQfcYom9nDY1OTBM3JbTOuqtIrzSIlFYdogyttXrTsmSue8TAoWWnHirKdkzeskIs5ZbI/pfGV3eRol+9Mn0MtmaoLp1VrAtSiZYwPRdcIGBDNMQ=
+  #     travis encrypt -r ubuntu-core/snappy COVERALLS_REPO_TOKEN=$secret_token
+  - secure: sc/JVLmOF6OESRSvPqRe3WFujYYbBX6FL0PH8jb6wUeALuqqFJPnb/5aLup0lCxCd+yJ0Puazo7EOX8Rk0txqmEYbufaPiPIxO3e5e32Md5qG5LX9JEUNiG1YAhB9IKqUeZBhWX6H3oOTCfko87j+NfJUOsqlKY4hWvYi4huGamrzIjLOK/lbPS6lSxaejEVACx8xUJax0KaVCu44JJQb1EfNjVkkhN374yFHfo5a8mBT1XwQCvkqXdPJ4phD4gFqqPy5zCrny3cKD10V8DhhhXtyDn71yP2saH4GXyPb9Uzv42n9hvafyZGElqNMgucpwgcnw1zBcHx3JvBye+MPJQhxAebMJ9kRbRsVp5xNh3YlcYw61st4aKutY5L3A42u+rlFzh+5I77ATj5XnPijd0IcHDCPMvaCoqQ2oQcBPtqHSYWMT0kVMA/u/THKwu8A6EtMWv55KNqd+ZW9Y5YHn5ODaAVi8rZby/QFLrMwrrO3Yq11VQFIB3ipafVNtTnAtACG4XkkxSiZ7U5eJQAX4B6Wn4anE9d+b3MzwKOD34pW9UNDqtA2KPutNdKRy6Pt7XAh+fPp9X71hNRegyxVNtqkTozaKCbbmfzZWb1bjC3esKBd9aLgjVy3IAQpQRklHWS6zyMNAXHFg29V16/K2d6KiD1zH+vPY7Kfmkfctc=
   matrix:
   - TEST_SUITE=static
   - TEST_SUITE=unit
   - TEST_SUITE=integration
-  - TEST_SUITE="examples --skip-install"
 
 install:
-  # PPA for plainbox.
-  - sudo apt-add-repository -y ppa:hardware-certification/public
+  - sudo apt-add-repository -y ppa:ubuntu-lxc/lxd-stable
   - sudo apt-get update -qq
-  - sudo apt-get install -qq build-essential squashfs-tools dpkg-dev plainbox pyflakes python3-apt python3-docopt python3-coverage python3-fixtures python3-jsonschema python3-mccabe python3-pep8 python3-requests python3-testscenarios python3-testtools python3-yaml python3-lxml
+  - sudo apt-get -o Dpkg::Options::="--force-confnew" -y dist-upgrade || true
+  - sudo apt-get install -qq lxd lxcfs || true
+before_script:
+  - sudo lxd-images import ubuntu xenial --stream daily --alias ubuntu
+  - sudo lxc launch ubuntu xenial -c security.privileged=true
```

Comment:
yup, but it didn't make any difference in the end. I was trying to use it to get rid of the chmod 777.
I still want to get rid of it, but I prefer to keep trying in a different PR.